packages/cli/api-importers/v2-importer-commons/src/converters/schema/SchemaConverter.ts

```diff
@@ -61,6 +63,91 @@ export class SchemaConverter extends AbstractConverter<AbstractConverterContext<
             }
         }
 
+        if (Array.isArray(this.schema.type)) {
+            return this.convertPrimitiveTypeArraySchema({ context, errorCollector });
+        }
+        return this.convertSchema({ context, errorCollector });
+    }
+
+    private async convertPrimitiveTypeArraySchema({
+        context,
+        errorCollector
+    }: {
+        context: AbstractConverterContext<object>;
+        errorCollector: ErrorCollector;
+    }): Promise<SchemaConverter.Output | undefined> {
+        if (this.schema.type == null || !Array.isArray(this.schema.type) || this.schema.type.length === 0) {
+            errorCollector.collect({
+                message: "Received unexpected schema type array type with no types.",
```

Comment:
Ideally there is a different error message for each of these conditions and a sample of what the user provided: 

- `Received invalid array schema []` 
- `Received non-array schema ${this.schema}`
- `Received null schema ${schema}`